Makefile

```diff
@@ -48,6 +48,14 @@ build: mod download-v3-binaries
 	@go build $(BUILD_FLAGS_MULTIPLEXER) -o build/celestia-appd ./cmd/celestia-appd
 .PHONY: build
 
+## build-skip-download: Build the celestia-appd binary into the ./build directory and skip downloading the v3 binaries.
+build-skip-download: mod
+	@cd ./cmd/celestia-appd
+	@mkdir -p build/
+	@echo "--> Building build/celestia-appd with multiplexer enabled"
+	@go build $(BUILD_FLAGS_MULTIPLEXER) -o build/celestia-appd ./cmd/celestia-appd
+.PHONY: build-skip-download
```

Comment:
rather than an additional target, it might cleaner to have

```sh
make build download=false
```

and have it default to true for running locally. WDYT?

Reply:
yea that works and seems preferable. Will update to your suggestion.